BibBuddy/app/src/main/java/de/bibbuddy/IsbnRetriever.java

```diff
@@ -37,18 +37,31 @@ private static Document loadXmlFromString(String xml) throws Exception {
     return builder.parse(inputSource);
   }
 
+  private String getField(Document xmlMetadata, String fieldName) {
+    String value = "";
+
+    try {
+      value = xmlMetadata.getElementsByTagName(fieldName).item(0).getTextContent();
+    } catch (Exception e) {
+      // exception for publication year, which needs an int
+      if (fieldName == "dcterms:issued") {
```

Comment:
```suggestion
      if (fieldName.equals("dcterms:issued")) {
```
String-values are not compared by "==" but by using equals().

Reply:
✓